src/test/java/com/google/firebase/messaging/MessageTest.java

```diff
@@ -678,6 +678,76 @@ public void testIncorrectAnalyticsLabelFormat() {
     }
   }
 
+  @Test
+  public void testImageInNotification() throws IOException {
+    Message message = Message.builder()
+        .setNotification(new Notification("title", "body", "image.png"))
```

Comment:
Use a URL test value. Define a constant at the top of the file and reuse it in the new test cases.